google/cloud/security/common/gcp_api/_base_client.py

```diff
@@ -41,8 +41,11 @@ def _attach_user_agent(request):
     Returns:
         HttpRequest: A modified googleapiclient request object.
     """
-    user_agent = request.headers['user-agent']
-    request.headers['user-agent'] = user_agent + ', %s/%s ' % (
+    user_agent = request.headers.get('user-agent', '')
+    if not user_agent or user_agent.contains(forseti_security.__package_name__):
```

Comment:
IOW: my question here is really why are we not adding a user-agent, when the request doesn't contain a user-agent in the first place.  Might be good to also add a comment to explain this in the future.  :)